client/src/components/Ratings.jsx

```diff
@@ -6,40 +6,77 @@ class Ratings extends React.Component {
   }
 
   render() {
-    return (
-      <ul>
-        <li>
-          <span>Cleanliness</span>
-          <span>-----------</span>
-          <span>Rating Number</span>
-        </li>
-        <li>
-          <span>Accuracy</span>
-          <span>-----------</span>
-          <span>Rating Number</span>
-        </li>
-        <li>
-          <span>Communication</span>
-          <span>-----------</span>
-          <span>Rating Number</span>
-        </li>
-        <li>
-          <span>Location</span>
-          <span>-----------</span>
-          <span>Rating Number</span>
-        </li>
-        <li>
-          <span>Check-in</span>
-          <span>-----------</span>
-          <span>Rating Number</span>
-        </li>
-        <li>
-          <span>Value</span>
-          <span>-----------</span>
-          <span>Rating Number</span>
-        </li>
-      </ul>
-    )
+    if(!this.props.averageRatings) {
+      return (
+        <ul>
+          <li>
+            <span>Cleanliness</span>
+            <span>-----------</span>
+            <span>Rating Number</span>
+          </li>
+          <li>
+            <span>Accuracy</span>
+            <span>-----------</span>
+            <span>Rating Number</span>
+          </li>
+          <li>
+            <span>Communication</span>
+            <span>-----------</span>
+            <span>Rating Number</span>
+          </li>
+          <li>
+            <span>Location</span>
+            <span>-----------</span>
+            <span>Rating Number</span>
+          </li>
+          <li>
+            <span>Check-in</span>
+            <span>-----------</span>
+            <span>Rating Number</span>
+          </li>
+          <li>
+            <span>Value</span>
+            <span>-----------</span>
+            <span>Rating Number</span>
+          </li>
+        </ul>
+      )
+    } else {
+      return (
+        <ul>
+          <li>
+            <span>Cleanliness</span>
+            <span>-----------</span>
+            <span>{this.props.averageRatings.cleanliness}</span>
+          </li>
+          <li>
+            <span>Accuracy</span>
+            <span>-----------</span>
+            <span>{this.props.averageRatings.accuracy}</span>
+          </li>
+          <li>
+            <span>Communication</span>
+            <span>-----------</span>
+            <span>{this.props.averageRatings.communication}</span>
+          </li>
+          <li>
+            <span>Location</span>
+            <span>-----------</span>
+            <span>{this.props.averageRatings.location}</span>
+          </li>
+          <li>
+            <span>Check-in</span>
+            <span>-----------</span>
+            <span>{this.props.averageRatings.checkIn}</span>
+          </li>
+          <li>
+            <span>Value</span>
+            <span>-----------</span>
+            <span>{this.props.averageRatings.value}</span>
+          </li>
+        </ul>
+      )
+    }
```

Comment:
same confusion here as with the avatar?

Reply:
I'm assuming you're asking why there isn't an avatar? this part is for the average of each individual rating type and a place holder for the rating bars